data/mods/DinoMod/monsters/zed-dinosaur.json

```diff
@@ -413,7 +459,8 @@
     "armor_cut": 19,
     "armor_bullet": 3,
     "hp": 1500,
-    "description": "Gigantic, long-necked, four-legged dinosaur corpse with a long neck and spiked tail protected by natural bone armor."
+    "description": "Gigantic, long-necked, four-legged dinosaur corpse with a long neck and spiked tail protected by natural bone armor.",
```

Comment:
Is the neck protected by the armor, the spiked tail, both, the entire dinosaur, or what?

Reply:
Unclear from the current fossil record. I'm going with the whole thing until the science is clearer. They looked like this maybe 
![image](https://user-images.githubusercontent.com/26608431/113496338-b79eb880-94c6-11eb-883a-4b1a4956b5d7.png)

Reply:
```suggestion
    "description": "Gigantic, long-necked, four-legged dinosaur corpse, protected by natural bone armor and a spiked tail.",
```
I just realized it said "long-necked" then mentioned a "long tail"... will check back for that!